aw_test.go

```diff
@@ -93,4 +104,74 @@ var _ = Describe("Airwave", func() {
 			})
 		})
 	})
+
+	Context("when gossiping", func() {
+		Context("when fully connected", func() {
+			It("should return content from all nodes", func() {
+				defer time.Sleep(time.Millisecond)
+
+				ctx, cancel := context.WithCancel(context.Background())
+				defer cancel()
+
+				// Initialise nodes.
+				n := 2
```

Comment:
Consider increasing this number so that we are actually testing the case of more than 2 nodes.